observability/cockpit/concepts.mdx

```diff
@@ -0,0 +1,174 @@
+
+## Alerting
+
+Alerting detects complex conditions defined by a rule and keeps you aware of issues in your environments. When a condition defined by a rule is met, the rule tracks it as an alert and responds by triggering one or more actions.
+
+</Concept>
+
+<Concept opened>
+
+## Alert manager
+
+Scaleway's alert manager allows you to manage and respond to alerts. It handles alerts sent when the alerting rules we run are firing. The alert manager triggers alerts (e.g. emails or texts) if a criteria you have configured on your applications' metrics and logs is activated.
+
+</Concept>
+
+<Concept>
+
+## Alerting rules
+
+Alerting rules allow you to define criteria that determine whether an alert is triggered. The rule consists of queries and expressions, a condition, the frequency of evaluation, and the duration over which the condition is met. When an alert is triggered, a notification is sent to the alert manager, which forwards the notification to receivers.
+
+</Concept>
+
+<Concept opened>
+
+## Cockpit
+
+A Cockpit is an instance of the Observability product that has a dedicated Grafana instance. A Scaleway Project can have only one Cockpit.
+
+</Concept>
+
+<Concept>
+
+## Contact points
+
+Contact points define how your contacts are notified when an alert fires. Contact point types include emails, Slack and texts. When an alert fires, a notification is sent to all contact point types listed for a contact point.
+
+</Concept>
+
+<Concept>
+
+## Endpoints
+
+An endpoint is the point of entry in a communication channel when two systems are interacting. The endpoint is the means from which the API can access the resources they need from a server to perform their task. An endpoint can include a URL of a server or service.
+
+</Concept>
+
+<Concept>
+
+## Loki Remote Write
+
+``Loki Remote Write`` is a feature that allows you to send metrics from almost any device to a Loki server.
+
+</Concept>
+
+<Concept>
+
+## LogQL
+
+LogQL is Grafana Loki’s language for querying logs. LogQL uses labels and operators for filtering.
+
+</Concept>
+
+<Concept>
+
+## Logs
+
+Logs provide a record of all events and errors that take place during the lifecycle of your resources. They represent an excellent source of visibility if you want to know when a problem occurred, or which events correlate with it.
+
+</Concept>
+
+<Concept>
+
+## Managed alerts
+
+Managed alerts are alert criteria that Scaleway defines for you. They allow you to receive alerts for behaviors that we deem abnormal on your products.
+
+</Concept>
+
+<Concept>
+
+## Managed dashboards
+
+A managed dashboard is a set of one or more panels organized into one or more rows where you can visualize your data.
+
+</Concept>
+
+<Concept>
+
+## Metric
+
+A metric is a lifecycle-related numerical representation of data (e.g. disk usage and CPU usage) measured over intervals of time. Metrics give you a bird's eye view of your infrastructure.
+
+</Concept>
+
+<Concept>
+
+## Monitoring
```

Comment:
Not sure this one is a good one, I doubt our visitor would not know what monitoring is about. Plus there are as many definitions as there are sysadmin 😄